src/components/dialogs/network-modifications/vsc/creation/vsc-creation-form.tsx

```diff
@@ -42,6 +42,7 @@ const VscCreationForm: FunctionComponent<VscCreationFormProps> = ({
                     currentNode={currentNode}
                     id={CONVERTER_STATION_1}
                     stationLabel={'converterStation1'}
+                    isModification={false}
```

Comment:
you may not write it, and use isModification for true value only (I'm not sure if you should change type isModification? in the ConverterStationPaneProps)

Reply:
do you mean make isModification = false by default ? so we don't need to specify it explicitly ?

Reply:
done